plasTeX/plastex

```diff
@@ -22,27 +23,34 @@ def collect_renderer_config(config):
         except ImportError as msg:
             continue
 
-        config += conf.config
+        conf.addConfig(config)
```

Comment:
I find the old behaviour is unintuitive. What we really are doing is that each Renderer contains a *section*, and we add the section to the existing config, whereas in the old system we need to first create a whole Config object and then somehow magically combine the two.

Reply:
Your explanation makes sense, but it doesn't come across in the code or documentation. Maybe we could actually only allow renderers to add config *sections*. But it may make sense for a renderer to add options to the files or images section. In this case I don't quite see the difference with "magically combining the two".

Reply:
In that case the behaviour would be explicit in the `addConfig` function instead of figuring out what addition exactly does (which isn't that complicated, to be fair)